apps/i18n/dance/en_us.json

```diff
@@ -18,24 +18,19 @@
   "danceAiModalHeading": "generate {input} effect",
   "danceAiModalChooseEmoji": "Choose three emojis.",
   "danceAiModalGenerateButton": "Generate",
-  "danceAiModalFinding": "A.I. is finding the best effect to generate.",
-  "danceAiModalFinding2": "A.I. is finding the best effect for your emojis.",
-  "danceAiModalGenerating": "A.I. is generating this effect.",
-  "danceAiModalGenerating2": "A.I. thinks this is the best effect for your emojis.",
+  "danceAiModalFinding": "A.I. is finding the best effect for your emojis.",
+  "danceAiModalGenerating": "A.I. thinks this is the best effect for your emojis.",
   "danceAiModalEffectButton": "Effect",
-  "danceAiModalEffect": "A.I. generated this effect.",
-  "danceAiModalEffect2": "A.I. thinks this is the best effect for your emojis and generated the code!",
+  "danceAiModalEffect": "A.I. thinks this is the best effect for your emojis and generated the code!",
```

Comment:
Will this rename to an existing key have an effect on our translations?

Reply:
Yeah I was hoping you'd know the answer to that :P @mgc1194 do you know if this is going to be problematic? Do I need to avoid re-using the string names? I wanted to keep the ones that don't have "2" at the end, but if that's going to cause problems with translations I can just clean up the unused ones and leave the "2" ones.